core-api/src/main/java/com/sequenceiq/cloudbreak/api/endpoint/v4/credentials/responses/InteractiveCredentialResponse.java

```diff
@@ -0,0 +1,42 @@
+package com.sequenceiq.cloudbreak.api.endpoint.v4.credentials.responses;
+
+import static com.sequenceiq.cloudbreak.doc.ModelDescriptions.EnvironmentRequestModelDescription.INTERACTIVE_LOGIN_CREDENTIAL_USER_CODE;
+import static com.sequenceiq.cloudbreak.doc.ModelDescriptions.EnvironmentRequestModelDescription.INTERACTIVE_LOGIN_CREDENTIAL_VERIFICATION_URL;
+
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+
+@ApiModel(value = "InteractiveCredential", description = "Contains values from an Azure interactive login attempt")
+public class InteractiveCredentialResponse {
+
+    @ApiModelProperty(value = INTERACTIVE_LOGIN_CREDENTIAL_USER_CODE, example = "B8ZT2QD4K")
```

Comment:
is B8ZT2QD4K a valid userCode or random?

Reply:
a modified valid one